src/openmrs-esm-module-config.ts

```diff
@@ -5,3 +5,5 @@ export {
 } from "./module-config/module-config";
 
 export { ModuleNameContext, useConfig } from "./react-hook/react-hook";
+
+export { validator } from "./validators/validators";
```

Comment:
Instead of exporting this function and causing everyone to have to use this function everywhere, I think we can actually accomplish 100% of the same behavior you're going for here without forcing users of esm-module-config to use a `validator` function at all!

See my comment inside of module-config.ts for how